neuralprophet/conformal.py

```diff
@@ -158,7 +159,30 @@ def _get_q_hat(self, df_cal: pd.DataFrame, noncon_scores: np.ndarray) -> float:
 
         return q_hat
 
-    def plot(self, plotting_backend: str):
+    def set_plotting_backend(self, plotting_backend):
```

Comment:
Do we need this method here, or can we call a util instead?

Reply:
Agree! Would move that into a new PR related to the general refactoring of the structure.